test/functional/DDR_Test/tck_ddrext.xml

```diff
@@ -43,8 +48,8 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<if>
 		<equals arg1="${OS}" arg2="os.zos"/>
 		<then>
-			<property name="DUMP_OPTION" value="-Xdump:system:events=vmstop,label=%uid.${dump.name},request=exclusive+compact+prepwalk -Xscmx16m -Xshareclasses:name=ddrextjunitSCC,addtestjithints -Xjit -Xaot:forceaot,count=5,disableAsyncCompilation -Xmx512M -version" />
-				<if>
+			<property name="DUMP_OPTION" value="-Xdump:system:defaults:file=${system.dump},request=exclusive+compact+prepwalk -Xscmx16m -Xshareclasses:name=ddrextjunitSCC,addtestjithints -Xjit -Xaot:forceaot,count=5,disableAsyncCompilation -Xmx512M" />
```

Comment:
This needs to use `label=` (or `dsn=`, not `file=`); z/OS does not support the `file=` sub-option.

Reply:
@Yuehan-Lin I believe this is the final point that I asked to be addressed: will you be pushing a change soon?

Reply:
Sure, I just pushed. I was just testing on vmfarm. And for my latest change, I changed `label=${system.dump}`back into `label=%uid.${dump.name}` in `test/functional/DDR_Test/tck_ddrext.xml`. 
This is because when I tested on vmfarm, teston zos gave the error
 ```
[exec] IEATDUMP failure for DSN='/HOME/J9BUILD/TMP/BLD_413603/TESTDDREXT_GENERAL_IBM_0/DDREXT.J9CORE.DMP.X&DS' RC=0x00000008 RSN=0x00000022
 [exec] JVMPORT023W IEATDUMP failed because user-specified dump template was too long. Retrying dump with default template.
```

Reply:
The special treatment of the definition of `DUMP_OPTION` afforded on z/OS will also need to be applied in `ddrext.setup.dump.dynamic`.

Reply:
I am not sure if this is correctly: do you mean that for `tck.config.dump.opt.1`, I should change it back to using `temp.tck.config.dump.opt` to discuss if os is zos or windows platform, then eventually set
```
<property name="tck.config.dump.opt.1" value="${temp.tck.config.dump.opt}"/>
```

Reply:
What I mean is that we need to effectively replace
```
<property name="system.dump.location" location="${system.dump}" />
```
with
```
<if>
  <isset property="isZOS" />
  <then>
    <property name="system.dump.location" value="${dump.name}${zos.core.suffix}" />
  </then>
  <else>
    <property name="system.dump.location" location="${system.dump}" />
  </else>
</if>
```
to avoid the path that is too long on z/OS.

Reply:
I just changed and pushed, please take a look

Reply:
That looks good. Have you confirmed that it addresses the problem you saw on z/OS?

Reply:
Not yet, it's still testing now. I will let you know the result once it's finished. Thank you!

Reply:
I changed playlist.xml to exclude `testDDRExt_General_ibm` on zos temporarily, since !vmcheck failed on zos platform, (runtimes/backlog/issues/124). And the rest ddr tests passed.